google/api_core/client_logging.py

```diff
@@ -0,0 +1,141 @@
+import logging
+import json
+import os
+
+from typing import List, Optional
+
+_LOGGING_INITIALIZED = False
+_BASE_LOGGER_NAME = "google"
+
+# Fields to be included in the StructuredLogFormatter.
+#
+# TODO(https://github.com/googleapis/python-api-core/issues/761): Update this list to support additional logging fields.
+_recognized_logging_fields = [
+    "httpRequest",
+    "rpcName",
+    "serviceName",
+]  # Additional fields to be Logged.
+
+
+# TODO(https://github.com/googleapis/python-api-core/issues/763): Expand documentation.
+def logger_configured(logger):
```

Comment:
```suggestion
def logger_configured(logger) -> bool:
```

We should also include the type of `logger`